ngs_mapper/tests/test_fasta_input.py

```diff
@@ -0,0 +1,31 @@
+import sh
+import unittest
+from glob import glob
+import tempfile
+from os.path import join, dirname
+
+def runsample(indir, outdir):
+    sh.runsample(indir, "tests/fixtures/functional/947.ref.fasta", "947", od=outdir)
+
+class TestFastaInput(unittest.TestCase):
+
+    def setUp(self):
+        self.fastaInputDir = tempfile.mkdtemp()
+        self.fastqInputDir = tempfile.mkdtemp()
+        self.fastaOutputDir = join(dirname(self.fastaInputDir), 'fastaout')
+        self.fastqOutputDir = join(dirname(self.fastqInputDir), 'fastqout')
+
+    def tearDown(self):
+       shutil.rmtree(self.fastaInputDir )
+       shutil.rmtree(self.fastqInputDir )
+       shutil.rmtree(self.fastaOutputDir)
+       shutil.rmtree(self.fastqOutputDir)
+
+    def test_run_stats_with_fasta_equals_run_with_fastq_40s(self):
+        runsample(self.fastaInputDir, self.fastaOutputDir)
+        runsample(self.fastqInputDir, self.fastaOutputDir)
```

Comment:
The test works by comparing the flagstats output between a run with fastq (all 40 quals) and fasta (all converted to fastq w/ 40 quals).

Except it doesn't work, because I forgot to copy the data over to the correct files. I made this PR a bit prematurely, maybe, as the work day was ending.